Training/PythonWebServer/index.html

```diff
@@ -9,6 +9,33 @@
     <script src="/resources/index.js" type="text/javascript"></script>
 </head>
 <body>
-    <h1>Hello world!</h1>
+	<div class='wrapper'>
+        <button class="button" onclick="loadDoc()">UPDATE</button>
+
+    	<div class="display">
+        Time:
+            <p id = "Time"></p> 
+
+        <br><br><br>
+    	</div>
+
+    	<div class="display">
+        Total CPUs: 
+        <p id = "totalCPU"></p>
+        <br> <br>
```

Comment:
It's better to use CSS to control spacing compared to using `<br>` tags, since different platforms/browsers might interpret `<br>` tags differently.